default.nix

```diff
@@ -1,2 +1,10 @@
-{ pkgs ? import (import ./nix/sources.nix).nixpkgs { } }:
-pkgs.callPackage ./github-deploy.nix { }
+(import
+  (
+    let lock = builtins.fromJSON (builtins.readFile ./flake.lock); in
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+      sha256 = lock.nodes.flake-compat.locked.narHash;
+    }
+  )
+  { src = ./.; }
+).defaultNix
```

Comment:
This makes it hard for non-flake users to inject the instance of nixpkgs they want to use. That's why I'm using a `flake.lock.nix`. See https://github.com/numtide/treefmt/tree/master

Maybe that `flake.lock.nix` could be added to flake-utils.